fix from upstream for building on systems w/out mmap (e.g. mingw)

From 6f1fbff8114ec2319dd874f3552105b378931eb9 Mon Sep 17 00:00:00 2001
From: Christos Zoulas <christos@zoulas.com>
Date: Fri, 15 Jul 2016 00:56:04 +0000
Subject: [PATCH] we only need mman for testing.

---
 src/der.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/der.c b/src/der.c
index fcdeb6c79814..fd093d29ee89 100644
--- a/src/der.c
+++ b/src/der.c
@@ -35,8 +35,6 @@
 #endif
 
 #include <sys/types.h>
-#include <sys/stat.h>
-#include <sys/mman.h>
 
 #include <stdio.h>
 #include <fcntl.h>
@@ -53,6 +51,8 @@
 #include "magic.h"
 #include "der.h"
 #else
+#include <sys/mman.h>
+#include <sys/stat.h>
 #include <err.h>
 #endif
 
-- 
2.9.0

